### ISSUE TITLE
avoid basicFlowFunctions to be inlined

### ISSUE THREAD

**Issue by mwetter** [ibpsa/modelica-ibpsa#1070]
This issue is to set `Inline=false` for the flow functions. JModelica does more aggressive optimization when compiling functions that are not inlined.

**Comment by mwetter** [comment]
For `IBPSA.Fluid.Examples.FlowSystem.Simplified6`, the simulation time in JModelica is about the same with and without inline:
```
$ git checkout issue1070_inline; for i in 1 2 3 4 5; do jm_ipython.sh jmodelica.py | grep Elapsed ; done; git checkout master; for i in 1 2 3 4 5; do jm_ipython.sh jmodelica.py | grep Elapsed ; done;
Already on 'issue1070_inline'
Elapsed simulation time: 0.0900130271912 seconds.
Elapsed simulation time: 0.0901470184326 seconds.
Elapsed simulation time: 0.0886640548706 seconds.
Elapsed simulation time: 0.0902850627899 seconds.
Elapsed simulation time: 0.0898900032043 seconds.
Switched to branch 'master'
Your branch is up to date with 'origin/master'.
Elapsed simulation time: 0.0906569957733 seconds.
Elapsed simulation time: 0.0885410308838 seconds.
Elapsed simulation time: 0.0891630649567 seconds.
Elapsed simulation time: 0.0885760784149 seconds.
Elapsed simulation time: 0.0894351005554 seconds.
```
The time for translation and simulation is slightly faster without inlining
```
$ git checkout issue1070_inline; for i in 1 2 3 4 5; do time jm_ipython.sh jmodelica.py | grep real ; done; git checkout master; for i in 1 2 3 4 5; do time jm_ipython.sh jmodelica.py | grep real ; done;
Switched to branch 'issue1070_inline'

real	0m18.106s
user	0m0.148s
sys	0m0.017s

real	0m17.967s
user	0m0.139s
sys	0m0.027s

real	0m18.245s
user	0m0.131s
sys	0m0.032s

real	0m18.132s
user	0m0.108s
sys	0m0.065s

real	0m18.000s
user	0m0.136s
sys	0m0.029s
Switched to branch 'master'
Your branch is up to date with 'origin/master'.

real	0m18.803s
user	0m0.126s
sys	0m0.032s

real	0m18.423s
user	0m0.136s
sys	0m0.023s

real	0m18.262s
user	0m0.132s
sys	0m0.028s

real	0m18.657s
user	0m0.100s
sys	0m0.045s

real	0m18.444s
user	0m0.136s
sys	0m0.028s
```
(total 90.37 vs 92.5 seconds).

Also, the intermediate formats such as Dymola's dsmodel.mof or JModelica's blt.html file are easier to read as the function calls are shown rather than expanded.

**Comment by Mathadon** [comment]
I tested the following example using Dymola:

```modelica
model test
  Real m_flow;
  Real dp;

equation 
  m_flow=sin(time/13.12348524);
  m_flow=IDEAS.Fluid.BaseClasses.FlowModels.basicFlowFunction_dp(
              dp=dp,
              k=1,
              m_flow_turbulent=1);

end test;
```

I removed the inverse flag to force an algebraic loop and then tested with/without inlining using `simulateModel("test", stopTime=10000000, method="Euler", fixedstepsize=1, resultFile="test");
`. This resulted in a computation times of ~3.92s which were not significantly different. 

While the computation time improvements are modest I see no reason not to accept this change.
